app/models/user.rb

```diff
@@ -11,6 +10,8 @@ class User < ApplicationRecord
   has_one :volunteer, dependent: :destroy
   has_and_belongs_to_many :department
 
+  before_validation :assign_primary_email, if: :profile
```

Comment:
Sorry for messing with that. I might have just accidently removed that when tests where blasting all the versions of red on me...